src/lib/fauna-client.mjs

```diff
@@ -150,9 +86,9 @@ export const runQueryFromString = (expression, argv) => {
  * @param {string} opts.apiVersion - The API version
  * @param {boolean} opts.extra - Whether to include extra information
  * @param {boolean} opts.color - Whether to colorize the error
- * @returns {object}
+ * @returns {Promise<string>}
  */
-export const formatError = (err, { apiVersion, extra, color }) => {
+export const formatError = async (err, { apiVersion, extra, color }) => {
```

Comment:
why change to async?

Reply:
colorize is async because `codeToAnsi` is async.

shiki has a sync highlighter, but I'd need to reimplement the codeToAnsi stuff...which I'll do in a follow.

Then I can remove all the asyncz.

Reply:
Async has been removed.